corehq/apps/app_manager/suite_xml/sections/menus.py

```diff
@@ -123,7 +126,12 @@ def domain_uses_usercase():
                         })
                         menu = Menu(**menu_kwargs)
 
-                    menu.commands.extend(get_commands())
+                    excluded_form_ids = []
+                    if root_module and root_module.doc_type == 'ShadowModule':
```

Comment:
can also do `if isinstance(root_module, ShadowModule):` (unless I'm missing the intention here), which has the advantage of being not just more succinct but would also show up if you did a Find Usages on `ShadowModule` (not just text search).